t/016-balancer.t

```diff
@@ -1,22 +1,14 @@
-use Test::Nginx::Socket 'no_plan';
-use Cwd qw(cwd);
+use lib 't';
+use TestAPIcast 'no_plan';
 
-my $pwd = cwd();
-my $apicast = $ENV{TEST_NGINX_APICAST_PATH} || "$pwd/apicast";
-
-$ENV{TEST_NGINX_LUA_PATH} = "$apicast/src/?.lua;;";
-$ENV{TEST_NGINX_HTTP_CONFIG} = "$apicast/http.d/*.conf";
-$ENV{TEST_NGINX_APICAST_PATH} = $apicast;
+$ENV{TEST_NGINX_HTTP_CONFIG} = "$TestAPIcast::path/http.d/*.conf";
 $ENV{RESOLVER} = '127.0.1.1:5353';
 
-
 env_to_nginx(
     'RESOLVER'
 );
 master_on();
-log_level('debug');
 repeat_each(1);
```

Comment:
Should we define `TEST_NGINX_REPEAT_EACH` instead?
From TestAPIcast: `repeat_each($ENV{TEST_NGINX_REPEAT_EACH} || 2);`

Reply:
Some tests force one iteration because 2nd one would hit cache and have different result.

Because the repeat_each is in the base this is an override. So all tests default to 2 (or the env var) except few ones that are forced to one.